src/main/webapp/app/exercises/shared/example-submission/example-submissions.component.html

```diff
@@ -27,6 +27,24 @@ <h4>
                 <tr>
                     <th><span jhiTranslate="global.field.id"></span></th>
                     <th><span jhiTranslate="artemisApp.exercise.title"></span></th>
+                    <th><span jhiTranslate="artemisApp.assessment.dashboard.columns.result"></span></th>
+                    <th>
+                        <span jhiTranslate="artemisApp.exampleSubmission.submissionSize"></span>
+                        <fa-icon
+                            *ngIf="exercise && exercise.type === exerciseType.TEXT"
+                            [icon]="'question-circle'"
+                            class="text-secondary ps-1"
+                            [ngbTooltip]="'artemisApp.exampleSubmission.textSubmissionSizeHint' | artemisTranslate"
+                            container="body"
+                        ></fa-icon>
+                        <fa-icon
+                            *ngIf="exercise && exercise.type === exerciseType.MODELING"
```

Comment:
[nit] Isn't it redundant to check for exercise, as exercise should always exist in the context of example submission dashboard?
I guess the check at L25 could be adjusted as well

Reply:
ah did not notice... thanks